Svg.Editor.Core.Tests/SvgDrawingCanvasTests.cs

```diff
@@ -105,7 +105,7 @@ public async Task CanAddElementAtScreenCenter()
             var child = children.Single();
             var b = child.GetBoundingBox(Canvas.GetCanvasTransformationMatrix());
 
-            var e = RectangleF.Create(365.359f, 224.3942f, 69.28204f, 51.21167f);
+            var e = RectangleF.Create(192.1539f, 224.3942f, 415.6923f, 51.21167f);
```

Comment:
changing expected value because its correct. Element still in center.
Apparently, I did not update the tests after commit ae452d66 the textspans rendered with a different boundary

Reply:
I also tested that by creating a branch before the commit and after to validate that it was this issue